install-wings.sh

```diff
@@ -220,15 +225,19 @@ function check_os_comp {
   fi
 
   virt_serv=$(virt-what)
-  if [ "$virt_serv" != "" ]; then
-    print_warning "Virtualization: ${virt_serv//$'\n'/ } detected."
-  fi
 
-  if [ "$virt_serv" == "openvz" ] || [ "$virt_serv" == "lxc" ] ; then # add more virtualization types which are not supported
-    print_warning "Unsupported type of virtualization detected. Please consult with your hosting provider whether your server can run Docker or not. Proceed at your own risk."
-    print_error "Installation aborted!"
-    exit 1
-  fi
+  case "$virt_serv" in
+    openvz | lxc)
+      print_warning "Unsupported type of virtualization detected. Please consult with your hosting provider whether your server can run Docker or not. Proceed at your own risk."
+      print_error "Installation aborted!"
+      exit 1
+      ;;
+    "")
+      ;;
+    *)
+      print_warning "Virtualization: ${virt_serv//$'\n'/ } detected."
```

Comment:
What's up with the `\n` at the end? I think I recall `virt_serv` output being extremely long.

Reply:
I think this doesn't make it go to a new line. (it removes \n from end). I copied this from internet on my first PR.

Reply:
```bash
$ export test="test"
$ echo "${test//$'\n'/ }"
test
$ export test="test\n\n\n"
$ echo "${test//$'\n'/ }"
test



``` 

Am I doing something wrong here? It does not remove the `\n`' for me.

Reply:
Ok tested on my system and seems that it doesn't do anything too. Will revert back normal.